### ISSUE TITLE
query about a performance problem of my Yarn based CaffeOnSpark system.

### ISSUE THREAD

**Issue by redburn82** [yahoo/CaffeOnSpark#94]
hello,

during last 2 weeks, i've been going to try setting up yarn based CaffeOnSpark to check a feasibility of distributed learning system.
so i have made it that system works with 2 old pc( i7 core, 8G ram, nvidia gtx660 and one has 500G ssd and other 2T hdd) on same local network.

but unlike my expectation, the case with 2 spark workers that is always slower than single spark worker.
although i've tried changing quite mamy parameters to get better performance, but all attemps were failed.

similarly #45 "training ImageNet On CaffeOnSpark" 's idea (more complex modes), 
i tried to make new lmdb for traing /test set from partial imagenet ilsvrc 2012 ( 10000 training data/800? test data ; because a lmdb from full data needs120~30G+temporally 120~30G and that is too big so this large data can't be managed on my testbed  )
but still each iteration with 2 workers was much slower than single worker testing.

i can't make heads or tails of this.
would you kindly please tell some kinds of hints for solveing this problem?
